build.sh

```diff
@@ -49,3 +48,13 @@ case "$os" in
     mkdir -p ${OUTDIR}
     cp -f build/bin/Datadog.Trace.ClrProfiler.Native.${SUFIX} ${OUTDIR}/OpenTelemetry.AutoInstrumentation.ClrProfiler.Native.${SUFIX}
 esac
+
+# build Managed
+cd $DIR
+OUTDIR="$( pwd )/src/Datadog.Trace.ClrProfiler.Native/bin/${BUILD_TYPE}/x64"
+
+mkdir -p "$OUTDIR/netstandard2.0"
+mkdir -p "$OUTDIR/netcoreapp3.1"
+
+dotnet publish -f netstandard2.0 -c ${BUILD_TYPE} src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed.csproj -o "$OUTDIR/netstandard2.0"
+dotnet publish -f netcoreapp3.1 -c ${BUILD_TYPE} src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed.csproj -o "$OUTDIR/netcoreapp3.1"
```

Comment:
Instead of doing the steps for each target framework, I suggest defining one variable on the script with the set of target frameworks according to the OS of the build machine:

- On Windows: `net452 net46 net461 netstandard2.0 netcoreapp3.1`
- Others: `netstandard2.0 netcoreapp3.1`

and then use the variable on a loop:
```suggestion
for framework in $sdkTargetFrameworks ; do
    dotnet publish -f $framework -c ${BUILD_TYPE} src/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed/OpenTelemetry.AutoInstrumentation.ClrProfiler.Managed.csproj -o "$OUTDIR/$framework"
done
```

If we eventually adopt the SDK we should invest in extracting the target from the SDK NuGet versions and adjust build and code accordingly. However, for the PoC manually changing according to the SDK package is good enough.

Reply:
The package used at https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/177#discussion_r638450690 doesn't have a version for `net46` so this is something that will need to be handled. For the moment I am tempted to remove `net46` from the SDK targets and dig deeper at this later.

Reply:
I have removed `net46` from the list